authorization-service/lambda/basicAuthorizer.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  APIGatewayAuthorizerResult,
+  APIGatewayRequestAuthorizerEvent,
+  Context,
+  Callback,
+} from "aws-lambda";
+
+export const handler = async (
+  event: APIGatewayRequestAuthorizerEvent,
+  context: Context,
+  callback: Callback<APIGatewayAuthorizerResult>
+) => {
+  if (!event.headers || !event.headers.Authorization) {
```

Comment:
Lambda handler will not be called if `Authorization` header is missing (if corresponding identity source is configured with request authorizer)